frontend/src/index.tsx

```diff
@@ -71,6 +71,9 @@ const Routes = () => {
 
 const Start: React.FC = () => {
   const [{ data, error, loading }] = useAxiosGet('/config');
+  // for debug the login issue on TEST env.
+  console.log(data, error, loading);
+
```

Comment:
This is going to debug the TEST env login issue(redirect to wrong KC). I will remove this once we see the console log.